src/helpers/isTokenUserOnline.js

```diff
@@ -0,0 +1,40 @@
+import {api} from 'helpers/api';
+
+const isDefined = (value) => {
+    return value !== undefined && value !== null;
+};
+
+async function callAPI(token, id) {
+    await api.get(`/users/${id}`).then(res => {
```

Comment:
use another API endpoint [MUST]

Reply:
why is that? shouldn't we be able to reuse the endpoints we already have?

Reply:
Well, because this API can be used by anyone registered. I guess you do not want to give the usertoken to anyone who asks for it

Reply:
+ the token is not passed in this endpoint